composer.json

```diff
@@ -39,7 +35,7 @@
         "zendframework/zend-paginator": "^2.7",
         "doctrine/doctrine-module": "^2.1",
         "doctrine/orm": "^2.5",
-        "doctrine/mongodb-odm": "^1.1",
+        "doctrine/mongodb-odm": "^2.0",
```

Comment:
`mongodb-odm` will provide own `DateTimeImmutable` type (named `date_immutable` in XML mapping files) in `2.1` (see https://github.com/doctrine/mongodb-odm/issues/1975).